src/lib/containers/entity/SchemaDrivenAnnotationEditor/template/ArrayFieldItemTemplate.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { ArrayFieldTemplateItemType } from '@rjsf/utils'
+
+/** The `ArrayFieldItemTemplate` component is the template used to render an items of an array.
+ *
+ * @param props - The `ArrayFieldTemplateItemType` props for the component
+ */
+export default function ArrayFieldItemTemplate<T, F>(
```

Comment:
Template that didn't exist before RJSF v5. Pulled from our custom ArrayFieldTemplate.